src/feed-input.scss

```diff
@@ -0,0 +1,78 @@
+@import "./new-settings";
+@import "./mixins";
+
+$fd-feed-input-row-height: 3rem;
+$fd-feed-input-row-max-height: 20rem;
+$fd-feed-max-width: calc(100vw * 2 / 3);
+
+$block: #{$fd-namespace}-feed-input;
+$thumb: #{$block}__thumb;
+$textarea: #{$block}__textarea;
+$submit-btn: #{$block}__submit-button;
+
+.#{$block} {
+  @include fd-reset();
+
+  display: flex;
+  flex-direction: row;
+  padding: 1rem 0.5rem 1rem 1rem;
+  background-color: #fff;
```

Comment:
You don't need this. We have a postcss taking care of this

Reply:
done